### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by teclator**
Update **rubocop** and auto-correct some of the cops. (Moved from https://github.com/yast/yast-s390/pull/93 to make the review of the PR easier)

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Offense count: 9
```
Comment (jreidinger) [inline .rubocop_todo.yml:1]: this looks a bad to have that todo file and especially when bunch of cops can be auto corrected like tabs one.

Comment (teclator) [reply .rubocop_todo.yml]: merged into the .rubocop.yml file and fixed the **Layout/Tab** cop

File: src/modules/DASDController.rb [hunk 2]
```diff
@@ -360,7 +357,7 @@ def Summary
     def probe_or_mock_disks
       mock_filename = ENV["YAST2_S390_PROBE_DISK"]
       if mock_filename
-        YAML.load(File.read(mock_filename))
+        YAML.safe_load(File.read(mock_filename))
```
Comment (jreidinger) [inline src/modules/DASDController.rb:360]: that is only tricky change. As it is for testing probe, do you try it? Sometimes if we export whole ruby object, then safe_load breaks it.

**Review by jreidinger — APPROVED** [review]
LGTM if you try that testing probing

File: src/modules/DASDController.rb (hunk 2 above)
Comment (teclator) [reply src/modules/DASDController.rb:360]: YAST2_S390_PROBE_DISK=test/data/probe_disk_dasd.yml rake run[dasd] works fine, but maybe it is safer to not change it by now, what do you think?

Comment (jreidinger) [reply src/modules/DASDController.rb:360]: if it works, lets keep change. It is safer and looks like we do not export any non trivial objects.

**Pull request merged**